### ISSUE TITLE
问一下这个是为什么呢?

### ISSUE THREAD

**Issue by citePerson** [jsksxs360/How-to-use-Transformers#28]
![image](https://github.com/user-attachments/assets/f5149372-73a4-4db7-972b-ad09394a18b0)

运行train_model_ner.py
loss_fn = nn.CrossEntropyLoss()
loss函数报错。
![image](https://github.com/user-attachments/assets/aea12bfe-cdda-4be7-975f-0fba5d95bdfb)
![image](https://github.com/user-attachments/assets/7886c4e9-640c-4b68-a016-02b61fd7d2c5)

**Comment by jsksxs360** [comment]
你好，经测试 train_model_ner.py 是可以正常运行的。你是否修改了代码？

特别注意 `loss = loss_fn(pred.permute(0, 2, 1), y)` 这一行，需要调换一下维度。

torch==2.0.1
transformers==4.34.0

**Comment by jsksxs360** [comment]
```
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
/Users/sxu/miniconda3/envs/inst/lib/python3.8/site-packages/transformers/optimization.py:411: FutureWarning: This implementation of AdamW is deprecated and will be removed in a future version. Use the PyTorch implementation torch.optim.AdamW instead, or set `no_deprecation_warning=True` to disable this warning
  warnings.warn(
Epoch 1/3
-------------------------------
loss: 1.614589:   0%|                        | 4/5216 [00:03<1:07:20,  1.29it/s]
```

**Comment by jsksxs360** [comment]
建议在 Linux 环境下运行代码，Windows 下运行可能会遇到各种问题。

新版 Windows 都自带 Linux 子系统，也可以在该系统下运行，如果不清楚的话可以搜索一下 WSL2

**Comment by SteafanMrZhou** [comment]
![Image](https://github.com/user-attachments/assets/777adf02-f31f-4054-8a3f-058707c7e150)
你需要转换 y 的类型为 long
